scripts/generate-imports/config.json

```diff
@@ -2,7 +2,8 @@
     "modules": {
         "core": ["global", "utility"],
         "combo": ["core", "lightbox-dialog", "form", "progress-spinner"],
-        "form": ["button", "checkbox", "field", "radio", "select", "switch", "textbox"]
+        "form": ["button", "checkbox", "field", "radio", "select", "switch", "textbox"],
+        "split-button": ["button", "menu-button"]
```

Comment:
@agliga Is this what you were thinking? A bundle? I was thinking more of just a new top level module that had a dependency on button and menu-button.